lib/render/editor/editor.html

```diff
@@ -65,12 +65,50 @@
           }
         };
 
+        const setTrashPosition = () => {
```

Comment:
@flbulgarelli I agree that we should keep everything in ECMA5 if possible. I think it'll be better if i open another issue for that because some arrow functions were introduced before this PR.